common-items.lic

```diff
@@ -292,7 +292,14 @@ module DRCI
   end
 
   def put_away_item?(item, container)
-    put_away_item_safe?(item, container)
+    if container.is_a?(Array)
+        container.each do |c|
+            return true if put_away_item_safe?(item, c)
+        end
+        return false
```

Comment:
Returning false is explicitly necessary else the method was returning the value of `container` 🤷‍♂️